src/components/Date/SimpleDate.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import Select from '../Select/Select';
+import NumberInput from '../NumberInput/NumberInput';
+import { months, days } from '../../helpers/options-for-select.js';
+
+const handleChange = (path, newValue, oldDate, onValueChange) => {
+  if (!onValueChange) return;
+
+  const date = {
+    ...oldDate,
+    [path]: newValue,
+  };
+
+  if (!date.month.value) {
+    date.day.value = '';
+  }
+
+  onValueChange(date);
+};
+
+// TODO: Refactor the date prop to be either a string or an object of strings
+
+/**
+ * A simple date picker with only display logic. For built-in validation, use the Date component.
+ */
+export const SimpleDate = ({
+  errorMessage,
+  date,
+  label,
+  name,
+  required,
+  ariaDescribedby,
+  minYear,
+  maxYear,
+  onValueChange,
+  onBlur,
+}) => {
+  const inputId = useRef(_.uniqueId('date-input-'));
+  const { day, month, year } = date;
+
+  let daysForSelectedMonth = [];
+  if (month.value) {
+    daysForSelectedMonth = days[month.value];
+  }
+
+  let errorSpanId;
+  let errorSpan = '';
+  let fieldsetClass;
+  if (errorMessage) {
+    fieldsetClass = 'input-error-date usa-input-error';
+    errorSpanId = `${inputId.current}-error-message`;
+    errorSpan = (
+      <span className="usa-input-error-message" role="alert" id={errorSpanId}>
+        <span className="sr-only">Error</span> {errorMessage}
+      </span>
+    );
+  }
+  const ariaDescribedbyWithError =
+    [errorSpanId, ariaDescribedby || ''].join(' ').trim() || undefined;
+
+  // If the active element is _not_ part of the SimpleDate, call onBlur
+  const blurIfOutsideField = () => {
+    // Push the function onto the event loop because otherwise,
+    // document.activeElement will be <body>, not the _actual_ active element.
+    setTimeout(() => {
+      if (
+        ![
+          `#${inputId.current} [name="${name}Month"]`,
+          `#${inputId.current} [name="${name}Day"]`,
+          `#${inputId.current} [name="${name}Year"]`,
+        ].some(
+          selector =>
+            document.querySelector(selector) === document.activeElement,
+        )
+      )
+        onBlur();
```

Comment:
Should we set a default function for the `onBlur` prop in order to do this?

Reply:
Oh, that's a good call. If `onBlur` isn't a function, this is going to throw an error. That error wouldn't be blocking anything since it's not evaluated as part of any other function's call stack thanks to `setTimeout`, but it's still no good.